apdu-core/src/command.rs

```diff
@@ -93,7 +102,15 @@ impl From<Command> for Vec<u8> {
         }
 
         if let Some(l) = le {
-            buffer.push(l);
+            if l > u8::MAX.into() {
+                if has_extended_le {
+                    buffer.push(0);
```

Comment:
Could you please use `0u8` instead to specify that is octet operation, as same as L94?